### DISCUSSION TITLE
loc idx out of lower bound

### DISCUSSION THREAD

**Discussion by lammps-vasp** [deepmodeling/deepmd-kit#515] [Q&A]
what do mean loc idx out of lower bound 

OMP: Info #191: KMP_AFFINITY: 2 sockets x 16 cores/socket x 1 thread/core (32 total cores)
OMP: Info #216: KMP_AFFINITY: OS proc to physical thread map:
OMP: Info #171: KMP_AFFINITY: OS proc 0 maps to socket 0 core 0
OMP: Info #171: KMP_AFFINITY: OS proc 1 maps to socket 0 core 1
OMP: Info #171: KMP_AFFINITY: OS proc 2 maps to socket 0 core 2
OMP: Info #171: KMP_AFFINITY: OS proc 3 maps to socket 0 core 3
OMP: Info #171: KMP_AFFINITY: OS proc 4 maps to socket 0 core 4
OMP: Info #171: KMP_AFFINITY: OS proc 5 maps to socket 0 core 5
OMP: Info #171: KMP_AFFINITY: OS proc 6 maps to socket 0 core 6
OMP: Info #171: KMP_AFFINITY: OS proc 7 maps to socket 0 core 7
OMP: Info #171: KMP_AFFINITY: OS proc 8 maps to socket 0 core 8
OMP: Info #171: KMP_AFFINITY: OS proc 9 maps to socket 0 core 9
OMP: Info #171: KMP_AFFINITY: OS proc 10 maps to socket 0 core 10
OMP: Info #171: KMP_AFFINITY: OS proc 11 maps to socket 0 core 11
OMP: Info #171: KMP_AFFINITY: OS proc 12 maps to socket 0 core 12
OMP: Info #171: KMP_AFFINITY: OS proc 13 maps to socket 0 core 13
OMP: Info #171: KMP_AFFINITY: OS proc 14 maps to socket 0 core 14
OMP: Info #171: KMP_AFFINITY: OS proc 15 maps to socket 0 core 15
OMP: Info #171: KMP_AFFINITY: OS proc 16 maps to socket 1 core 0
OMP: Info #171: KMP_AFFINITY: OS proc 17 maps to socket 1 core 1
OMP: Info #171: KMP_AFFINITY: OS proc 18 maps to socket 1 core 2
OMP: Info #171: KMP_AFFINITY: OS proc 19 maps to socket 1 core 3
OMP: Info #171: KMP_AFFINITY: OS proc 20 maps to socket 1 core 4
OMP: Info #171: KMP_AFFINITY: OS proc 21 maps to socket 1 core 5
OMP: Info #171: KMP_AFFINITY: OS proc 22 maps to socket 1 core 6
OMP: Info #171: KMP_AFFINITY: OS proc 23 maps to socket 1 core 7
OMP: Info #171: KMP_AFFINITY: OS proc 24 maps to socket 1 core 8
OMP: Info #171: KMP_AFFINITY: OS proc 25 maps to socket 1 core 9
OMP: Info #171: KMP_AFFINITY: OS proc 26 maps to socket 1 core 10
OMP: Info #171: KMP_AFFINITY: OS proc 27 maps to socket 1 core 11
OMP: Info #171: KMP_AFFINITY: OS proc 28 maps to socket 1 core 12
OMP: Info #171: KMP_AFFINITY: OS proc 29 maps to socket 1 core 13
OMP: Info #171: KMP_AFFINITY: OS proc 30 maps to socket 1 core 14
OMP: Info #171: KMP_AFFINITY: OS proc 31 maps to socket 1 core 15
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9243 thread 0 bound to OS proc set 0
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9243 thread 1 bound to OS proc set 16
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9252 thread 2 bound to OS proc set 1
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9253 thread 3 bound to OS proc set 17
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9254 thread 4 bound to OS proc set 2
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9255 thread 5 bound to OS proc set 18
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9256 thread 6 bound to OS proc set 3
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9257 thread 7 bound to OS proc set 19
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9258 thread 8 bound to OS proc set 4
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9259 thread 9 bound to OS proc set 20
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9260 thread 10 bound to OS proc set 5
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9261 thread 11 bound to OS proc set 21
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9262 thread 12 bound to OS proc set 6
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9263 thread 13 bound to OS proc set 22
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9265 thread 15 bound to OS proc set 23
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9264 thread 14 bound to OS proc set 7
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9266 thread 16 bound to OS proc set 8
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9267 thread 17 bound to OS proc set 24
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9268 thread 18 bound to OS proc set 9
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9269 thread 19 bound to OS proc set 25
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9270 thread 20 bound to OS proc set 10
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9271 thread 21 bound to OS proc set 26
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9272 thread 22 bound to OS proc set 11
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9274 thread 24 bound to OS proc set 12
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9273 thread 23 bound to OS proc set 27
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9275 thread 25 bound to OS proc set 28
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9276 thread 26 bound to OS proc set 13
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9277 thread 27 bound to OS proc set 29
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9278 thread 28 bound to OS proc set 14
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9279 thread 29 bound to OS proc set 30
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9280 thread 30 bound to OS proc set 15
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9281 thread 31 bound to OS proc set 31
OMP: Info #252: KMP_AFFINITY: pid 8346 tid 9282 thread 32 bound to OS proc set 0
# warning: loc idx out of lower bound (ignored if warned for more than 10 times)
# warning: loc idx out of lower bound (ignored if warned for more than 10 times)
# warning: loc idx out of lower bound (ignored if warned for more than 10 times)
# warning: loc idx out of lower bound (ignored if warned for more than 10 times)
# warning: loc idx out of lower bound (ignored if warned for more than 10 times)
# warning: loc idx out of lower bound (ignored if warned for more than 10 times)
# warning: loc idx out of lower bound (ignored if warned for more than 10 times)
# warning: loc idx out of lower bound (ignored if warned for more than 10 times)
# warning: loc idx out of lower bound (ignored if warned for more than 10 times)
# warning: loc idx out of lower bound (ignored if warned for more than 10 times)

**Comment by njzjz** [comment]
This is because your atom is on the boundary. Usually, you can ignore this warning.

**Comment by amcadmus** [comment]
see #21
